src/test/java/core/basesyntax/services/implementation/ValidatorImplTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.services.implementation;
+
+import core.basesyntax.services.Validator;
+import org.junit.Test;
+
+public class ValidatorImplTest {
+    private static final String VALUE_WITH_WRONG_LENGTH = "b,apple,red,20";
+    private static final String VALUE_WITH_WRONG_QUANTITY = "b,apple,-20";
+    private static final Validator validator = new ValidatorImpl();
+
+    @Test (expected = RuntimeException.class)
+    public void shouldThrowRunTimeExceptionWhenValueNull_Not_Ok() {
```

Comment:
Better naming for tests:
`methodUnderTest_state_expectedBehavior`
for example `read_fileNotExist_notOk`